app/code/Magento/Checkout/Test/Unit/CustomerData/ItemPoolTest.php

```diff
@@ -91,19 +99,19 @@ public function testGetItemDataIfItemExistInMap()
     }
 
     /**
-     * @expectedException \Magento\Framework\Exception\LocalizedException
      * #@expectedExceptionMessage product_type doesn't extend \Magento\Checkout\CustomerData\ItemInterface
```

Comment:
If you have a chance I would suggest removing this line as well. It's pretty confusing since it contains the annotation.

Reply:
Thanks. Will handle that. This needs to be removed definitely.